src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -345,6 +345,69 @@ public virtual Task PrepareAsync()
                 TaskScheduler.Default);
         }
 
+        /// <summary>
+        /// Prepares the cancellable media player for playback, asynchronously.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token to cancel preparing.</param>
+        /// <seealso cref="CancellationToken"/>
+        /// <returns>A task that represents the asynchronous prepare operation.</returns>
+        /// <remarks>To prepare the player, the player must be in the <see cref="PlayerState.Idle"/> state,
+        /// and a source must be set.
+        /// The state must be <see cref="PlayerState.Preparing"/> to cancel preparing.
+        /// When preparing is cancelled, a state will be changed to <see cref="PlayerState.Idle"/> from <see cref="PlayerState.Preparing"/>.
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">
+        ///     Operation failed; internal error.
+        ///     -or-<br/>
+        ///     The player is not in the valid state.
+        ///     </exception>
+        /// <seealso cref="PrepareAsync()"/>
+        /// <seealso cref="Unprepare()"/>
+        /// <seealso cref="Prepared"/>
+        /// <since_tizen> 6 </since_tizen>
+        public async Task PrepareAsync(CancellationToken cancellationToken)
```

Comment:
It's not `virtual` method otherwise existing `public virtual Task PrepareAsync()` method.
Is it intended?

Reply:
as far as i know, virtual keyword allows it to be overridden. will it be needed for it?
i also wonder the reason why PrepareAsync() has a virtual keyword.